packages/atomic/src/pages/v1.html

```diff
@@ -113,6 +113,10 @@
         margin: 0;
       }
 
+      body.atomic-modal-opened {
+        overflow-y: hidden;
+      }
```

Comment:
Could we manage this style inside the modal component?

Reply:
It's on the body 😄 To remove the scrollbar, it has no link with the modal itself

Reply:
I think clients would always want to have this style to avoid dual scroll bars when the modal is opened, no?
I think it's nicer when components look good OOTB, with needing to tweak and adjust these details,

Reply:
I don't like forcing a style on the customer's page, outside of Atomic, and sometimes the container could be different than the body. For these reason's I wouldnt do this for now